### ISSUE TITLE
Postgres SQL Interpolation

### ISSUE THREAD

**Issue by amerryma** [zeroturnaround/sql-formatter#119]
Hi, been using the postgres formatting and was running in to an issue with sql interpolation. See [here](https://www.postgresql.org/docs/9.5/app-psql.html#APP-PSQL-INTERPOLATION).

### Given

```sql
SELECT * FROM my_table WHERE my_table.my_value = :value;
```

with options:

```js
{
    language: 'postgresql',
    uppercase: true
}
```

## Actual Output

```sql
SELECT
  *
FROM
  my_table
WHERE
  my_table.my_value =: VALUE;
```

## Expected Output

```sql
SELECT
  *
FROM
  my_table
WHERE
  my_table.my_value = :value;
```

Note the retaining of the case, as well as the position of the `:`. 

Let me know if I can provide more information, and if this is a simple configuration issue, I'd love some help!

**Comment by nene** [comment]
Looks like the `:name` placeholders had been left out of the implementation. Should be fixed in `4.0.2` release.

Regarding the `uppercase` option. It's not really that safe to use, as the parser isn't that good at always telling keywords and identifiers apart. Though that again depends on whether the identifiers are case-sensitive or not.

**Comment by amerryma** [comment]
Thanks for the quick response! I'll be sure not to use uppercase for now, thank you.
